package.json

```diff
@@ -93,6 +93,7 @@
     "md5": "^2.3.0",
     "papaparse": "~5.3.0",
     "phovea_clue": "github:phovea/phovea_clue#develop",
+    "plotly": "^1.0.6",
```

Comment:
Why do we need the "[Plotly Node API](https://www.npmjs.com/package/plotly)" as dependency here?

```suggestion
```

Reply:
we dont, removed